mmpose/core/visualization/image.py

```diff
@@ -174,14 +177,16 @@ def imshow_keypoints_3d(
 
     for idx, res in enumerate(pose_result):
         kpts = res['keypoints_3d']
+        valid = kpts[:, 3] >= kpt_score_thr
+
```

Comment:
Consider using kpts = kpts[valid] to simplify the code.

Reply:
If I use kpts = kpts[valid], the limb also need to change.